MODULE.bazel

```diff
@@ -0,0 +1,71 @@
+module(name = "rules_ros", repo_name = "com_github_mvukov_rules_ros")
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "bzip2", version = "1.0.8.bcr.1")
+bazel_dep(name = "eigen", version = "3.4.0.bcr.1")
+bazel_dep(name = "gflags", version = "2.2.2")
+bazel_dep(name = "glog", version = "0.7.0")
+bazel_dep(name = "googletest", version = "1.14.0.bcr.1")
+bazel_dep(name = "lz4", version = "1.9.4")
+bazel_dep(name = "platforms", version = "0.0.10")
+bazel_dep(name = "rules_cc", version = "0.0.9")
+bazel_dep(name = "rules_foreign_cc", version = "0.10.1")
+bazel_dep(name = "rules_python", version = "0.33.1", dev_dependency=True)
+
+non_module_ros_repositories = use_extension("//extensions:ros_repositories.bzl", "non_module_dependencies")
+use_repo(
+    non_module_ros_repositories,
+    "console_bridge",
+    "roscpp_core",
+    "rosconsole",
+    "ros_genmsg",
+    "ros_gencpp",
+    "ros_genpy",
+    "ros_std_msgs",
+    "ros_comm_msgs",
+    "ros_comm",
+    "ros_ros",
+    "ros_common_msgs",
+    "ros_actionlib",
+    "ros_dynamic_reconfigure",
+    "ros_geometry2",
+    "orocos_kdl",
+    "urdfdom_headers",
+    "tinyxml",
+    "urdfdom",
```

Comment:
This is the ugly part, but nothing we can do about that. I wish there was a way to import a variable from a repo.